src/app/loginComponents/requests/requests.component.ts

```diff
@@ -115,6 +127,19 @@ export class RequestsComponent extends Base implements OnInit {
     });
   }
 
+  openDeleteDialog(name: string, id: number): void {
+    const dialogRef = this.dialog.open(OrganizationDeleteConfirmDialogComponent, {
+      width: '400px',
```

Comment:
No pixels :)

Reply:
Fixed, openDeletetDialog() is now deleted as the organization deletion function now uses deregister-modal.